rpm/qtimageformats.spec

```diff
@@ -78,25 +77,31 @@ find %{buildroot}%{_libdir}/cmake/Qt5Gui/ -type f -name "*_*Plugin.cmake" \
 
 %files plugin-mng
 %defattr(-,root,root,-)
+%license LICENSE.LGPLv21 LGPL_EXCEPTION.txt LICENSE.LGPLv3
```

Comment:
I think LGPLv3 needs GPLv3 terms too so these probably should include also LICENSE.GPLv3.

> This version of the GNU Lesser General Public License incorporates
> the terms and conditions of version 3 of the GNU General Public
> License, supplemented by the additional permissions listed below.

Reply:
Duh. Guess so.

Reply:
Done